Resources/translations/FOSMessageBundle.es.yml

```diff
@@ -1,22 +1,27 @@
-messenger: Mensajes
-inbox: Carpeta de entrada
-sent: Enviado
-send_new: Enviar nuevo mensaje
-search: Buscar
-threads_found: %num% hilo encontrado | %num% hilos encontrados
+messenger:      Mensajes
+inbox:          Carpeta de entrada
+sent:           Enviado
+deleted:        Eliminados
+send_new:       Enviar nuevo mensaje
+search:         Buscar
+threads_found:  %num% hilo encontrado | %num% hilos encontrados
 
-message_info: Por %sender%, el %date%
-reply: Responder
+message_info:   Por %sender%, el %date%
+reply:          Responder
```

Comment:
Please revert the alignment. It is a pain for maintenance (adding a longer key would then force to edit all lines of the file, which is impossible to review).
I'm not able to know whether you are doing an actual change or no when seeing this diff.